drivers/entropy/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-zephyr_library()
-
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_ESP32_RNG          entropy_esp32.c)
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_MCUX_RNGA          entropy_mcux_rnga.c)
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_MCUX_TRNG          entropy_mcux_trng.c)
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_NRF5_RNG           entropy_nrf5.c)
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_SAM_RNG            entropy_sam.c)
-zephyr_library_sources_ifdef(CONFIG_ENTROPY_STM32_RNG          entropy_stm32.c)
-zephyr_library_sources_ifdef(CONFIG_FAKE_ENTROPY_NATIVE_POSIX  fake_entropy_native_posix.c)
-zephyr_library_sources_ifdef(CONFIG_USERSPACE                  entropy_handlers.c)
+zephyr_sources_ifdef(CONFIG_ENTROPY_ESP32_RNG          entropy_esp32.c)
```

Comment:
What is the difference between from zephyr_library_sources_ifdef and zephyr_sources_ifdef?

Reply:
zephyr_library() creates a library using cmake function add_library() and then adds sources to it by calling target_sources(${LIBRARY_NAME} PRIVATE ${source_file}) through zephyr_library_sources... function. 

zephyr_sources...() adds sources to "zephyr" target like this: target_sources(zephyr PRIVATE ${source_file})